blogposts/2024-07-18-running-rnw-from-a-mac.md

```diff
@@ -0,0 +1,86 @@
+---
+post_title: "Building React Native for Windows Apps from a MacOS Device"
```

Comment:
Right now, this feels like "My steps for building an RNW app: a case study"

I like the original title, I'd recommend reframing to focus on readers rather than author.